services/bstats/bstats-players.service.js

```diff
@@ -0,0 +1,68 @@
+'use strict' // (1)
```

Comment:
Can you remove these inline numbered comments (`// (1)`, etc.)

Reply:
Forgot about that, give me a moment.

Reply:
no worries!